lore/end_users/status/README.md

```diff
@@ -40,3 +46,7 @@ The Zuul Pipelines counter provides an easy look at how many pipelines exist in
 ### Service Status
 
 Nodepool, Zookeeper, and Zuul are three core services that are essential to the stability of a BonnyCI cluster. If anything except the green "OK" message is displayed for these service statuses, then BonnyCI could be experiencing a full service outage.
+
+## Build Status
+
+The [Build Status page](https://travis-ci.org/BonnyCI) links to the Travis-CI page with our repositories. Yes we see the humor in a CI tool using a different CI tool for testing. This is a temporary home for us.
```

Comment:
@mlangbehn I don't believe zuul/BonnyCI offers a status page that shows what you're looking for. All it ever shows is the jobs that are currently running, it does not show the state of a repo.